_posts/2020-01-20-memkind-dax-kmem.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Memkind support for KMEM DAX option
+author: michalbiesek
+layout: post
+identifier: memkind-support-for-KMEM-DAX-option
+---
+
+### Introduction
+
+Linux kernel version 5.1 brings in [support][kernel-patch] for the volatile-use of persistent memory
+as a hotplugged memory region (**KMEM DAX**).
+From the perspective of OS, persistent memory will be seen as a separate NUMA node(s).
+[libmemkind][memkind-release] API was extended to include new kinds that allow for automatic detection and allocation
+from these new persistent memory NUMA nodes.
+
+### Requirements
+
+<span>1.</span> Kernel 5.1 with **KMEM DAX** driver enabled:
+
+To configure proper driver installation run `nconfig` and enable driver.
```

Comment:
You don't need to compile your own kernel (which this sentence only implies) if support for KMEM DAX is already compiled in, so this sentence should reflect that.